openfecwebapp/api_caller.py

```diff
@@ -86,14 +86,19 @@ def load_legal_advisory_opinion(ao_no):
     if not (documents and len(documents)):
         return None
 
-    canonical_document = documents[0]
+    for document in documents:
+        canonical_document = document
+        if (document['category'] == 'Final Opinion'):
+            break        
+    
```

Comment:
In the case of none of the documents matching `Final Opinion`, going with the category of the final document will suffice?

Also,  can you guarantee the case here?  Would it be better to call `lower()` on the `document['category']` value and compare it to `final opinion`?